src/LibraryManager/Helpers/SemanticVersion.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 
-namespace Microsoft.Web.LibraryManager.Providers.Unpkg
+namespace Microsoft.Web.LibraryManager.Helpers
```

Comment:
>Helpers [](start = 39, length = 7)

Move this to its own folder named something like 'Versioning'

Reply:
Create SemanticVersion Folder

---
In reply to: [203548469](https://github.com/aspnet/LibraryManager/pull/324#discussion_r203548469) [](ancestors = 203548469)